### PR TITLE
wip: start wrapped_email.html

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Comment by lesleyjanenorton** [comment]
Adds this to the top of forwarded emails. String needs some work 

Screen Shot 2020-05-14 at 12 34 29 PM

File: emails/templates/emails/wrapped_email.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            <tr>
+              <td align="center">
+                <p class="relay-text" style="color: #4a4a4f; font-size: 14px; line-height: 150%; font-family: sans-serif;">This email was sent to your alias <span style="font-weight: bolder; color: #6200a4;">{{ display_email|safe }}</span>. To stop receiving emails sent to this alias, update the forwarding settings in your dashboard.</p>
+              </td>
+            </tr>
+            <tr>
+              <td width="100%" valign="center" align="center" style="text-align: center; padding-top: 20px;">
+                <a
+                  class="visit-dashboard"
+                  href="http://127.0.0.1:8000/accounts/profile/?utm_source=emails&utm_medium=email&utm_campaign=alias-email"
```
Comment (groovecoder) [inline emails/templates/emails/wrapped_email.html:61]: This needs to be a dynamic host/domain.

File: emails/templates/emails/wrapped_email.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    </style>
+  </head>
+  <body style="padding: 0; margin: 0;">
+    <table width="100%" bgcolor="#f9f9fa" style="background: #f9f9fa; padding-top: 0; padding-right: 30px; padding-left: 30px; padding-bottom: 40px; margin-top: 0px; margin-bottom: 30px; width: 100%;">
+      <tr>
+        <td align="center" valign="top" width="100%" style="width: 100%; max-width: 550px; padding-top: 20px;">
+          <table width="100%" style="text-align: center; border-collapse: collapse; max-width: 550px;">
+            <tr>
+              <td align="center" style="margin-top: 0">
+                <img width="100%" src="http://127.0.0.1:8000/static/images/private-relay-beta-logo-emails.png" style="display: inline-block; margin-bottom: 0px; width:100%; max-width: 280px;" alt="Private Relay Logo" />
```
Comment (groovecoder) [inline emails/templates/emails/wrapped_email.html:49]: Needs to be dynamic host/domain src value.

**Comment by groovecoder** [comment]
I added `SITE_ORIGIN` env var, django settings, and template context var to make the email links dynamic per site. I think this is ready to go?

**Comment by TyDraniu** [comment]
This commit broke plain text e-mails I receive. Now there's only this big header and the 'None' word below. Text message is missing, only new html part show up.

**Pull request merged**